includes/admin/settings/register-settings.php

```diff
@@ -898,6 +898,12 @@ function edd_get_registered_settings() {
 		)
 	);
 
+	if ( ! edd_shop_supports_buy_now() ) {
+		$edd_settings['misc']['button_text']['buy_now_text']['disabled']      = true;
```

Comment:
Any reason you're adding these disabled input fields instead of graying out the `Button Behavior` option on the Download metabox?